docs/sql/scalar.txt

```diff
@@ -101,6 +105,92 @@ which defines the length of the substring::
     SELECT 1 row in set (... sec)
 
 
+.. _scalar_char_length:
+
+char_length('string') returns integer
+-------------------------------------
+
+The ``char_length`` returns number of characaters in the string.
+
+synopsis
+........
+
+::
+
+    CHAR_LENGTH (str)
```

Comment:
the other functions so far don't have a synopsis, do we really need one for the new ones?